iChor/iChorCNA_step2/iChorCNA_step2.wdl

```diff
@@ -21,16 +25,18 @@ task bundle_plots {
 }
 
 workflow ichorSummaryBundle {
-    Array[File] CNA_plots
-    File summary_script
-    String sample_set_name
+    input {
+        Array[File] CNA_plots
+        File summary_script
+        String sample_set_name
+    }
 
     call bundle_plots {
         input: CNA_plots = CNA_plots,
                summary_script = summary_script,
                sample_set_name = sample_set_name
     }
     output {
-        bundle_plots.*
+        File bundledPDF = bundle_plots.bundledPDF
```

Comment:
There's only this one output from `bundle_plots`, so I just kept the same name from the task